website/src/components/icons-list.tsx

```diff
@@ -72,11 +74,28 @@ const IconsList = () => {
     >
       {Object.keys(iconList).map((key, i) => {
         const Icon = iconList[key]
+        const { onCopy } = useClipboard(key)
```

Comment:
I think the correct text should be `<PhoneIcon />`,

Reply:
It was my intention but after duscussing (https://github.com/chakra-ui/chakra-ui/issues/3889#issuecomment-826138438) I think it's better to copy only the name, because the name you can copy directly in the `import {...` and on the tags